practices/code_coverage.md

```diff
@@ -0,0 +1,58 @@
+#Code Coverage
+
+##Short Description
+Code coverage measures the percentage of source code lines that are covered by automated tests.
+
+##Tags
+-
+-
+-
+-
+
+##Keywords
+- coverage
+- library
+- reports
+- code lines
+- function
+- branch
+- test
+
```

Comment:
@Kristynaslomkova This structure isn't correct. See the structure of e.g. User Stories: https://github.com/DXHeroes/knowledge-base/pull/49/files

The meta info must begin with `---` and end with the second `---`. After that is the content of the article.